docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -49,7 +49,10 @@ means that you need to supply a labeled training dataset that has some
 {feature-vars} and a {depvar}. The {classification} algorithm learns the 
 relationships between the features and the {depvar}. Once you’ve trained the 
 model on your training dataset, you can reuse the knowledge that the model has 
-learned about the relationships between the data points to classify new data.
+learned about the relationships between the data points to classify new data. 
+Your training dataset should be balanced, otherwise the {classanalysis} may not 
```

Comment:
This is a bit strong as currently worded IMO, we handle imbalanced training data up to a point
```suggestion
Your training dataset should be approximately balanced, otherwise the {classanalysis} may not 
```